src/_gettsim/sozialversicherung/rente/altersrente/für_frauen/altersgrenze.yaml

```
@@ -0,0 +1,344 @@
+      10: 63.916667
+      11: 64.0
+      12: 64.0
+    1951:
+      1: 64.083333
+      2: 64.083333
+      3: 64.166667
+      4: 64.166667
+      5: 64.25
+      6: 64.25
+      7: 64.333333
+      8: 64.333333
+      9: 64.416667
+      10: 64.416667
+      11: 64.5
+      12: 64.5
+    1952:
+      1: 64.583333
+      2: 64.583333
+      3: 64.666667
+      4: 64.666667
+      5: 64.75
+      6: 64.75
+      7: 64.833333
+      8: 64.833333
+      9: 64.916667
+      10: 64.916667
+      11: 65.0
+      12: 65.0
+  1996-09-27:
+    reference: Wachstums- und Beschäftigungsförderungsgesetz 1996. BGBl. I S. 1461
+    note: Increase of FRA accelerated with exemption (Vertrauensschutz).
+    entry_age_old_regime: 60.0
+    entry_age_new_regime: 65.0
+    max_birthyear_old_regime: 1939
+    min_birthyear_new_regime: 1945
+    1940:
+      1: 60.083333
+      2: 60.166667
+      3: 60.25
+      4: 60.333333
+      5: 60.416667
+      6: 60.5
+      7: 60.583333
+      8: 60.666667
+      9: 60.75
+      10: 60.833333
+      11: 60.916667
+      12: 61.0
+    1941:
+      1: 61.083333
+      2: 61.166667
+      3: 61.25
+      4: 61.333333
+      5: 61.416667
+      6: 61.5
+      7: 61.583333
+      8: 61.666667
+      9: 61.75
+      10: 61.833333
+      11: 61.916667
+      12: 62.0
+    1942:
+      1: 62.083333
+      2: 62.166667
+      3: 62.25
+      4: 62.333333
+      5: 62.416667
+      6: 62.5
+      7: 62.583333
+      8: 62.666667
+      9: 62.75
+      10: 62.833333
+      11: 62.916667
+      12: 63.0
+    1943:
+      1: 63.083333
+      2: 63.166667
+      3: 63.25
+      4: 63.333333
+      5: 63.416667
+      6: 63.5
+      7: 63.583333
+      8: 63.666667
+      9: 63.75
+      10: 63.833333
+      11: 63.916667
+      12: 64.0
+    1944:
+      1: 64.083333
+      2: 64.166667
+      3: 64.25
+      4: 64.333333
+      5: 64.416667
+      6: 64.5
+      7: 64.583333
+      8: 64.666667
+      9: 64.75
+      10: 64.833333
+      11: 64.916667
+      12: 65.0
+altersgrenze_für_frauen_vorzeitig:
+  name:
+    de: Altersgrenze für vorzeitigen Bezug der Altersrente für Frauen.
+    en: Early retirement age for women.
+  description:
+    de: >-
+      § 237a SGB VI
+      Frühestmögliches Alter ab dem die Altersrente für Frauen bezogen werden kann
+      (mit Abschlägen).
+      https://www.sozialgesetzbuch-sgb.de/sgbvi/237a.html
+    en: >-
+      § 237a SGB VI
+      Earliest possible age to receive pension for women (with deductions).
+      https://www.sozialgesetzbuch-sgb.de/sgbvi/237a.html
+  unit: Years
+  reference_period: null
+  type: require_converter
+  1980-01-01:
+    value: 60.0
+  1989-12-18:
+    reference: Rentenreformgesetz 1992. BGBl. I S. 2261 1989 § 41
+    note: Increase of early retirement age from 60 to 62 for birth cohort 1949-1952.
+    entry_age_old_regime: 60.0
+    entry_age_new_regime: 62.0
+    max_birthyear_old_regime: 1948
+    min_birthyear_new_regime: 1953
+    1949:
+      1: 60.083333
```

Unless we change this to piecewise_linear, I'd suggest:
```suggestion
      1: 
        years: 60
        months: 1
```

More verbose, but clearer it seems?

---

Yes, all Altersgrenzen should be like this eventually. I haven't touched Frauen, langjährig and besonders langjährig yet (just created the files)